### DISCUSSION TITLE
command: atri build ssg Error in windows os

### DISCUSSION THREAD

**Discussion by Arv-78** [Atri-Labs/atrilabs-engine#407] [Help: Installation & Start] [answered]
the web app is running on a local environment , but while uploading it on gihub page gives error while building the app.

![Screenshot (34)](https://user-images.githubusercontent.com/83110137/196037478-3fc0892b-e03e-475e-aa64-1f5d1f548c52.png)

**Comment by darshitac11** [comment, marked as answer]
@Arv-78 - Thanks for raising this problem. 

This is an open issue and we are working on it. #366
